tests/IdentifyTest.php

```diff
@@ -3,14 +3,69 @@
 namespace Unicodeveloper\Identify\Test;
 
 use PHPUnit_Framework_TestCase;
+use Sinergi\BrowserDetector\Browser;
+use Sinergi\BrowserDetector\Device;
+use Sinergi\BrowserDetector\Language;
+use Sinergi\BrowserDetector\Os;
+use Unicodeveloper\Identify\Identify;
```

Comment:
Since we are solely using PHP 7, you can take advantage of it by making this just one line of code like so:

```php

use Sinergi\BrowserDetector\{ Browser, Device, Language, Os };

```